src/lithium/__init__.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# coding=utf-8
+# flake8: noqa
+# pylint: disable=missing-docstring
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import lithium.interestingness.crashes
+# import lithium.interestingness.diff_test  # diff_test is not used outside of Lithium
+import lithium.interestingness.hangs
+import lithium.interestingness.outputs
+import lithium.interestingness.range
+import lithium.interestingness.timed_run
+import lithium.interestingness.utils
```

Comment:
I think I see what you're trying to do, but the right place for this is `interestingness/__init__.py`:

    from . import crashes, hangs, outputs, timed_run, utils
    #from . import diff_test  # diff_test is not used outside of Lithium
    from . import range  # pylint: disable=redefined-builtin

Reply:
You're right!

Note from truber over IRC:

"doing it in lithium/\_\_init\_\_.py doesn't directly accomplish it, it's actually a side-effect. you're actually importing lithium.interestingness.* into the lithium namespace, so that lithium.lithium.interestingness.crashes would work too"